### DISCUSSION TITLE
cooperation?

### DISCUSSION THREAD

**Discussion by mrozpara** [gillett-hernandez/Foreman2#3] [Ideas]
Hi, 
are you interested in further development and adding new functions to Foreman2? I've created own fork and added few new functions:
- scrolling with WSAD keys
- output/source while holding shift key (like ctrl and pass-through)
- list of files on lefthandside
- tabs (multiple files open at the same time)
- graph summary on rightside (not as modal window) + search functions
- mark assembler as done (usefull while building large structures to mark what is ready)
etc.
I can clean my code and prepare it as pull requests - if you are interested...

**Comment by gillett-hernandez** [comment]
Hi! I'm interested, let's talk more on discord. You should be able to find me in the py discord server.
